dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Validation31TestServlet.java

```diff
@@ -70,6 +82,10 @@ public void nestedRecordsTest() throws Exception {
         // TODO test that cascade validation occurs by creating a record which contains another record class
         // (For example a Person that has an Email address) then confirm that it validates both.
         // See section 5.6.4 of the Bean Validation specification
+
+        Employee emp = new Employee(null, new EmailAddress("emp1@gmail.com"));
```

Comment:
Avoid using real domains like gmail, this is probably a real email address, and since we don't control it we don't want it in our code. A safe domain to use is `example.com` since it's a reserved domain for use in documentation (so `emp1@example.com` would be fine)